lang/java/reef-common/src/main/java/org/apache/reef/io/WorkingDirectoryTempFileCreator.java

```diff
@@ -40,7 +40,7 @@
 
   @Inject
   WorkingDirectoryTempFileCreator() throws IOException {
-    this.tempFolderAsFile = new File("./reef/temp");
+    this.tempFolderAsFile = new File("./target/reef/temp");
```

Comment:
No, I don't think so. This is for temporary files created by the driver or evaluator. Those file should reside withing in `./reef/` hierarchy.